### DISCUSSION TITLE
Start - End Overlapping Resouce

### DISCUSSION THREAD

**Discussion by zipus** [frePPLe/frepple#575] [Q&A]
Why it's that in my resource detail plan I have gaps and overlapping start and end dates?
![imagen](https://user-images.githubusercontent.com/1032986/201717004-8fbc7157-2bd0-4dcf-8a0a-e46f6213c60c.png)

See gaps in tasks 5356 & 5391 it ends at 8:17, but next task doesn't start after 5 min later at 8:22? Why?
See overlapping in tasks 7844 & 5704, how it is possible to have two things at a time in a default resource?
Shouldn't a task precede the next one?
My resource is defined as:

```
name | "Afilado HERCAT"
type | "default"
maximum | "1.00000000"
available | "calendar"
source | "production.work.center,2"
lastmodified | "2022-11-14 18:07:29.251338"
overloadcount | "0"
```
With calendar and calendar interval as:
```
{"name":"calendar", "defaultvalue":"0E-8", "source":"Tryton"}
```
```
{"id":"6",
"calendar":"calendar",
"startdate":"1971-01-01 00:00:00",
"enddate":"2030-12-31 00:00:00",
"value":"1.00000000", "priority":"0",
"monday":"True", "tuesday":"True", "wednesday":"True", "thursday":"True", "friday":"True",
"saturday":"False", "sunday":"False",
"starttime":"08:00:00", "endtime":"17:59:59",
"lastmodified":"2022-11-14 18:03:42.573201"}
```

**Comment by zipus** [comment]
I still haven't found any hint about this, and it's really frustrating as the resources should limited (employee in the example) and can only be doing one task at time.
![imagen](https://user-images.githubusercontent.com/1032986/213193630-068bdae7-131d-4e81-85b4-3f8f372dd0d1.png)

Any hint?

**Reply by zipus** [reply]
Should I be setting the resource as constrained to true? What is this feature for? The definition on the docs is not enough for me.
Also this part on the definition says: `The default is true, except for resources of type infinite.` but when I go through UI their value for the resource is displayed as unknown.

Thanks.

**Reply by jdetaeye** [reply]
A blank value in the field is interpreted as the default value "true"

**Reply by jdetaeye** [reply]
If the status of the overlapping MO is "confirmed", that's expected behavior. The scheduler takes "confirmed" MO as fixed and frozen, even if they are infeasible.

If the status of the overlapping MO is "approved", then it's expected behavior in the Community Edition.  In the Enterprise Edition, they are rescheduled to be non-overlapping.

If the status of the overlapping MO is "proposed" then there is something wrong in your model. Neither in the Community Edition or Enterprise Edition shall we generate MO that are violating the capacity constraints (provided the resource is marked "constrained" and you ran a capacity constrained plan). 

In your first screenshot the status was proposed so your model must have had something wrongly configured.  In your second screenshot I don't see the status but I recognize from the reference format that it's an externally imported approved or confirmed MO.
